Leanplum-SDK/Classes/LPAppIconManager.m

```diff
@@ -82,6 +82,18 @@ + (void)uploadAppIconsOnDevMode
  */
 + (BOOL)supportsAlternateIcons
 {
+    // Run on main thread.
+    if (![NSThread isMainThread]) {
+        BOOL __block value = NO;
```

Comment:
better naming for `value`

Reply:
i changed to outputValue... Not sure what else to name.